README.md

```diff
@@ -2,69 +2,34 @@
 
 ## Setup
 
-Clone this repository on the target VM:
+Run the following command with values provided to you for your account:
 
 ```
-# May need to install git first
-# e.g. in RHEL
-# > sudo yum check-update
-# > sudo yum install git -y
-
-git clone https://github.com/lupfoss/lupmgr.git
-
-cd lupmgr
-```
-
-Run the following command with the value as provided by Lightup:
-
-```
-echo 'export LIGHTUP_CUSTOMER_TLA=<value>' > user_config.sh
-```
-
-From the repository folder, run the first part of the setup sequence based on the OS you are using:
-```
-RHEL7
-./rhel7_setup_part_one.sh
-
-RHEL8
-./rhel8_setup_part_one.sh
-
-Ubuntu 20.04
-./setup_part_one.sh
+LIGHTUP_TLA=<tla> LIGHTUP_TOKEN=<token> bash -c $(curl -L https://raw.githubusercontent.com/lupfoss/lupmgr/onecmd/bootstrap.sh)
```

Comment:
As discussed on slack, lightup-token could contain the TLA?

Reply:
as discussed, ignoring for now.